.buildkite/pipeline.yml

```diff
@@ -28,7 +28,12 @@ steps:
   - label: "🔬 Validate Podspec"
     key: "validate"
     command: |
-      validate_podspec
+      # validate_podspec
+      echo '+++ ⚠️ validate_podspec was bypassed ⚠️'
```

Comment:
It might be worth adding a link to the `.buildkite/validate_podspec_annotation.md` documentation from here.